src/cluster-legacy.c

```diff
@@ -4753,7 +4643,7 @@ void clusterCron(void) {
         /* If we are a master and one of the slaves requested a manual
          * failover, ping it continuously. */
         if (server.cluster->mf_end &&
-            nodeIsMaster(myself) &&
+                clusterNodeIsMaster(myself) &&
```

Comment:
nit: Weird indentation here
```suggestion
            clusterNodeIsMaster(myself) &&
```

Reply:
fixed